### ISSUE TITLE
Should there be a warning when trying to create a recipe that already exists?

### ISSUE THREAD

**Issue by isabelrios** [fxbox/app#117]
In foxbox logs we can see:

[2016-04-22 10:55:45.352] INFO  [thinkerbell@link.mozilla.org] No need to create a new service for this rule; ID 'Motion Sensor detects motion, light gets turned on.' already exists.

But in the UI there is no warning, you can go ahead with the flow to create a recipe and tap on Done and even taken back to recipes list view as if the recipe was created, but it is not.

What do you think?
Thanks!

**Comment by azasypkin** [comment]
Yeah, I think we should. But I don't see any difference in foxbox response.

Hey @mcav can\should we have specific response in case rule hasn't been added (because the one with the same id already exists)? If not, I think we can handle this on the client side as well.
